### PR TITLE
Feature/es5 bundle

### PR DESCRIPTION
**Pull request by michielvandergeest**

### CODE REVIEW

File: devtools/lightning-inspect.es5.js [hunk 1]
```diff
@@ -0,0 +1,825 @@
+(function (global, factory) {
```
Comment (g-zachar) [inline devtools/lightning-inspect.es5.js]: Build results should not be version controlled. Instead, I can update npm package with es5 bundles.

Comment (michielvandergeest) [reply devtools/lightning-inspect.es5.js]: @g-zachar the dist folder (and automatic build step upon each commit with husky) is added intentionally to make sure the built versions are available even if you npm install straight from GitHub

Comment (g-zachar) [reply devtools/lightning-inspect.es5.js]: That's true, but regular developer should use npm registry package in the first place. Install from github should be only in special cases e.g. in your case, where you are actively contributing, but in this case having to rebuild bundles each time shouldn't be that much of a problem.

**Comment by wouterlucas** [comment]
@g-zachar and @michielvandergeest lets park the dist vs npm discussion, I'll schedule something to talk about that including @woutermeek and @erikhaandrikman (and anyone that wants to chime in).

For now let's focus on ES5 support as that's quite high prio and touches a few components more. This is one of the first.

**Pull request merged**